.github/workflows/CI.yml

```
@@ -23,12 +23,20 @@ jobs:
           - ubuntu-20.04
           - macos-12
           - macos-11
-        node:
-          - 16
-        pnpm:
-          - 7
         cache_reset_counter:
           - 0
+        node:
+          - 12
+          - 16
+          - 18
+        include:
+          - node: 12
+            pnpm: 6
+            skip_test: true
+          - node: 16
+            pnpm: 7
+          - node: 18
+            pnpm: 8
```

We don't need to test with different pnpm versions. Please just specify the latest as the version

```suggestion
        pnpm:
          - 8
```